docs/test-merges.md

```diff
@@ -0,0 +1,28 @@
+== bin/test-merges.js documentation ==
```

Comment:
FWIW, this isn't Markdown (or at least GitHub is ignoring it).
Not sure if you want `## bin/test-merges.js documentation` (for an H2) or something else.
Or this is fine too, it's just docs.